docs/interfaces/commands-and-shells.rst

```diff
@@ -4,28 +4,25 @@
  Commands and Shells
 #####################
 
-In addition to structured model training workloads handled using :ref:`experiments <experiments>`,
-Determined also supports free-form tasks using *commands* and *shells*. Commands and shells enable
-you to use a Determined cluster and cluster GPUs without writing code that conforms to the trial
-APIs.
+Determined Commands and Shells provide support for running a Determined cluster without writing a
+model. This article describes how to manage GPU-powered batch commands and interactive shells.
```

Comment:
nit: seems funny to call this an article? maybe "this document"?

Reply:
resolved